src/js/pages/services/DeploymentsTab.js

```diff
@@ -152,10 +152,13 @@ class DeploymentsTab extends mixin(StoreMixin) {
   renderStatus(prop, deployment) {
     const title = `Step ${deployment.getCurrentStep()} of ${deployment.getTotalSteps()}`;
     const services = deployment.getAffectedServices();
+    const action = deployment.currentActions.map(function (action) {
+      return action.action;
+    }).join(', ');
     const items = services.map(function (service, index) {
       return (
         <li key={index}>
-          {service.getStatus()}
+          {service.getStatus()} ({action})
```

Comment:
action would never be null in that case. But it is not error safe if currentActions isn't present it would fail I changed this way to a reduce. And an if in the map below.